Source/Plugins/Core/com.equella.core/src/com/tle/integration/lti/brightspace/BrightspaceLtiWrapperExtension.java

```diff
@@ -59,6 +60,20 @@ public String getUserId(HttpServletRequest request) {
     return null;
   }
 
+  @Override
```

Comment:
I'd like to ignore the custom parms for Canvas and D2L lti signons, with the hope to bring all LTI signons over to the generic flow eventually.

Reply:
Already answered